readthedocsext/theme/templates/includes/elements/promotions/security-logs.html

```diff
@@ -0,0 +1,9 @@
+<div class="ui card">
+  <div class="content">
+    <a class="header" target="_blank" href="https://docs.readthedocs.io/page/security-log.html">Secure your organization with security logs</a>
```

Comment:
Redundant wording here

```suggestion
    <a class="header" target="_blank" href="https://docs.readthedocs.io/page/security-log.html">Monitor your organization with security logs</a>
```